apps/user-office-frontend/src/i18n.ts

```diff
@@ -0,0 +1,27 @@
+import i18n from 'i18next';
+import Backend from 'i18next-http-backend';
+import { initReactI18next } from 'react-i18next';
+
+i18n
+  .use(Backend)
+  .use(initReactI18next)
+  .init({
+    debug: true,
```

Comment:
I am also wondering what this debug: true really does and if it should be hardcoded like this.

Reply:
Missed that from test thanks!